sauna/consumers/ext/http_server/html.py

```diff
@@ -0,0 +1,80 @@
+import datetime
+
+from . import HTTPServerConsumer
+
+
+def get_html():
+    checks = HTTPServerConsumer.get_checks_as_dict()
+    checks_html = ''
+    for check_name, data in sorted(checks.items()):
+        date = datetime.datetime.fromtimestamp(
+            data['timestamp']
+        ).strftime('%Y-%m-%d %H:%M:%S')
+        checks_html += "<tr>"
+        checks_html += "<td>{}</td><td><span class=\"st st_{}\">{}</span>" \
+                       "</td><td>{}</td>" \
+                       "<td>{}</td>".format(
+                            check_name, data['code'], data['status'],
+                            data['output'], date
+                        )
+        checks_html += "</tr>"
+    return """<html>
+<head>
```

Comment:
Could you use HTML5 and define the encoding as UTF-8? I get `Temperature okay (46Â°C)`.